go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/gofiber/fiber/v3
 go 1.20
 
 require (
+	github.com/gofiber/fiber/v2 v2.52.0
```

Comment:
You should be able to remove these with go mod tidy

Reply:
I think those need to be removed @ReneWerner87

Reply:
sure thx